security/auth_throttling/__init__.py

```diff
@@ -31,10 +31,12 @@ def delay_message(remainder):
     """
     # TODO: There's probably a library for this.
     minutes = round(float(remainder) / 60)
-    return (_("1 minute")             if minutes == 1 else
-            _("%d minutes") % minutes if minutes > 1 else
-            _("1 second")             if ceil(remainder) == 1 else
-            _("%d seconds") % ceil(remainder))
+    return (
```

Comment:
We should probably just change this to a regular if / else block. I don't know why co-worker thought this was a more straight forward way to do this. Making it PEP8 doesn't really fix the underlying clarity issue.

```
if minutes == 1:
    return _("1 minute")
elif minutes > 1:
    return _("%d minutes") % minutes
...
...
```